css/style.css

```
@@ -0,0 +1,340 @@
+    -webkit-background-clip: text;
+}
+
+/******** section4 ********/
+
+/* todo : 배경 사이즈 크기가 맞지 않음 */
+.savelink.img {
+    background: url("/images/img1.png"), lightgray 50% ;
+    border-radius: 25px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 170%;
+    box-shadow: 0px 4px 25px 0px rgba(0, 0, 0, 0.08);
+}
+
+.savelink.img .dim {
+    position: relative;
+    width: 570px;
+    height: 489px;
+    background: rgba(0, 0, 0, 0.40);
+    border-radius: 25px;
+    box-shadow: 0px 4px 25px 0px rgba(0, 0, 0, 0.08);
+}
+
+.savelink.img .dim .popup {
+    position: absolute;
+    width: 416px;
+    height: 271px;
+    flex-shrink: 0;
+    top: 121px;
+    left: 77px;
+}
+
+.share {
+    background: linear-gradient(90deg, #6D7DCE, #93A8C9);
+    color: transparent;
+    background-clip: text;
+    -webkit-background-clip: text;
+}
+
+/******** section5 ********/
+.search.img {
+    overflow: hidden;
+    position: relative;
+    width: 550px;
+    height: 450px;
+    border-radius: 15px;
+    background-color: var(--grayF0);
+}
+
+.search.img img {
+    position: absolute;
+    width: 619px;
+    height: 584px;
+    background: linear-gradient(180deg, rgba(255, 255, 255, 0.00) 48.72%, #000 128.17%);
+    top: 55px;
+    left: 60px;
+    flex-shrink: 0;
+    border-radius: 15px;
+    box-shadow: 0px 4px 25px 0px rgba(0, 0, 0, 0.08);
+}
+
+.search.img .dim
+{
+    position: absolute;
+    width: 619px;
+    height: 584px;
+    background: linear-gradient(180deg, rgba(255, 255, 255, 0.00) 48.72%, #000 128.17%);
+    top: 55px;
+    left: 60px;
+    flex-shrink: 0;
+    border-radius: 15px;
+    box-shadow: 0px 4px 25px 0px rgba(0, 0, 0, 0.08);
+}
+
+.search.text {
+    display: inline;
+    background: var(--fandom-k-gra-blue-to-pink, linear-gradient(271deg, #99BAD8 -9.84%, #68E8F9 107.18%));
+    color: transparent;
+    background-clip: text;
+    -webkit-background-clip: text;
+}
+
+/******** footer ********/
+footer {
+    height: 280px;
+    border-radius: 30px;
+    padding-top: 120px;
+    box-shadow: 0px 0px 16px 0px #00000005;
+}
+
+footer .footer.background {
+    display: flex;
+    height: 160px;
+    padding: 32px 104px 64px 104px;
+    flex-direction: column;
+    align-items: center;
+    flex-shrink: 0;
+    align-self: stretch;
+    background-color: #111322;
+}
+
+.footer.content {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    align-self: stretch;
+}
+
+.name {
+    display: flex;
+    color: #676767;
+    text-align: center;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+    font-family: Arial;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 18px;
+    letter-spacing: 0em;
+}
+
+.option {
+    display: flex;
+    width: 181px;
+    color: #cfcfcf;
+    text-align: left;
+    font-family: Arial;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 18px;
+    letter-spacing: 0em;
```

0 은 단위를 작성하지않고, 0으로 할 수 있어요. (다른 스타일도 모두 동일합니다.)

```suggestion
    letter-spacing: 0;
```

그런데 이 부분을 letter-spacing 을 설정하신 이유가 있을까요? 따로 설정하지 않아도 동일할 것 같아서요!

(지금 보니 이 부분은 1주차때도 올려주셨던 코드였군요. Github 으로 보니까 좀 더 명확하게 보여서 추가로 리뷰드리는 점 양해 부탁드릴게요.)

---

처음에 피그마에 설정된 CSS대로 무조건 해야되는줄 알고 하다보니까 넣었던거 같습니다 😅
이 부분은 삭제하도록 하겠습니다!